core/domain/story_fetchers.py

```diff
@@ -321,6 +323,73 @@ def get_story_summaries_by_ids(
     return story_summaries
 
 
+def get_learner_group_syllabus_story_summaries(
+    story_ids: List[str]
+) -> List[story_domain.LearnerGroupSyllabusStorySummaryDict]:
+    """Returns the learner group syllabus story summary dicts
+    corresponding the given story ids.
+
+    Args:
+        story_ids: list(str). The list of story ids for which the story
+            summaries are to be returned.
+
+    Returns:
+        list(LearnerGroupSyllabusStorySummaryDict). The story summaries
+        corresponds to given story ids.
+    """
+    # If story check is applied only for mypy type checks. All story ids are
```

Comment:
The first sentence is hard to parse. Can you rewrite it? Maybe put quotes around "if story", or something.

Reply:
Updated this comment.